src/plugins/condition/ConditionManager.js

```diff
@@ -140,7 +141,7 @@ export default class ConditionManager extends EventEmitter {
     }
 
     cloneCondition(conditionConfiguration, index) {
-        this.createAndSaveCondition(index, JSON.parse(JSON.stringify(conditionConfiguration)));
+        this.createAndSaveCondition(index, Object.assign({}, conditionConfiguration));
```

Comment:
Why did this get changed to Object.assign? That won't create a deep copy. JSON.parse does.